general/okina.hpp

```diff
@@ -36,15 +40,31 @@
 template <size_t BLOCKS, typename DBODY, typename HBODY>
 void wrap(const size_t N, DBODY &&d_body, HBODY &&h_body)
 {
-   const bool gpu = mfem::config::usingGpu();
-   if (gpu)
-   {
-      return cuWrap<BLOCKS>(N,d_body);
-   }
-   else
+   const bool gpu = mfem::config::usingGpu();   //
+   const bool raja = mfem::config::usingRaja(); //
+   if(gpu && raja)
    {
-      for (size_t k=0; k<N; k+=1) { h_body(k); }
+#ifdef MFEM_USE_RAJA
+     printf("RAJA enabled! \n");
+     return RAJA::forall<RAJA::cuda_exec<BLOCKS>>(RAJA::RangeSegment(0,N), d_body);
+#else
+     MFEM_ABORT("RAJA requested for MFEM but RAJA is not enabled!");
+#endif
+
+  }else if (gpu) {
```

Comment:
There are some minor formatting issues here through line 65. This should get taken care of with a `make style`.

Reply:
Thanks! I have to admit I'm not familiar with make style though....  Is it supported on the LLNL machines? If I just run make style I get the following: 
``$ make style
/bin/sh: astyle: command not found
No source files were changed.``

Reply:
In order to use `make style`, you need `astyle` installed and in your shell's `PATH` environment variable. Unfortunately, I am not familiar with the LLNL desktops. If it's Red Hat Enterprise Linux, then you may be able to install it with the yum package manager...

Reply:
Cool thanks!